packages/@uppy/companion/src/server/provider/zoom/adapter.js

```diff
@@ -2,7 +2,7 @@ const moment = require('moment')
 
 const MIMETYPES = {
   MP4: 'video/mp4',
-  M4A: 'audio/mp4',
+  M4A: 'audio/m4a',
```

Comment:
I think this should be mp4

Reply:
you are right, I have fixed it. Thank you for catching it!